### ISSUE TITLE
[BUG] <Error: No such option: --quality>

### ISSUE THREAD

**Issue by Gabri04** [nathom/streamrip#640]
### Describe the bug

The quality option doesn't seem to work.

`rip url --quality 3 https://tidal.com/browse/album/147569387
Usage: rip url [OPTIONS] URLS...
Try 'rip url --help' for help.

Error: No such option: --quality`

### Command Used

```bash
rip url --quality 3 https://tidal.com/browse/album/147569387
```

### Debug Traceback

```text
rip -vvv url --quality 3 https://tidal.com/browse/album/147569387
[11:08:11] DEBUG    Showing all debug logs                                                                    cli.py:102
Usage: rip url [OPTIONS] URLS...
Try 'rip url --help' for help.

Error: No such option: --quality
```

### Config File

```toml
it didn't find one

`Error: No such option: --open`
```

### Operating System

Windows

### streamrip version

2.0.5

**Comment by apostoiis** [comment]
For me, it doesn't work with `--codec` flag either. So it must be something more generic?

**Comment by Gabri04** [comment]
I agree with you, it has to be a more generic problem

**Comment by CriticalPeriod** [comment]
-m -c options work in version 1.9.7

**Comment by AllKind** [comment]
```
rip --help
Usage: rip [OPTIONS] COMMAND [ARGS]...

  Streamrip: the all in one music downloader.

Options:
  --version                    Show the version and exit.
  --config-path PATH           Path to the configuration file
  -f, --folder DIRECTORY       The folder to download items into.
  -ndb, --no-db                Download items even if they have been logged in
                               the database
  -q, --quality INTEGER RANGE  The maximum quality allowed to download
                               [0<=x<=4]
  -c, --codec TEXT             Convert the downloaded files to an audio codec
                               (ALAC, FLAC, MP3, AAC, or OGG)
  --no-progress                Do not show progress bars
  -v, --verbose                Enable verbose output (debug mode)
  --help                       Show this message and exit.

Commands:
  config    Manage configuration files.
  database  View and modify the downloads and failed downloads databases.
  file      Download content from URLs in a file.
  id        Download an item by ID.
  lastfm    Download tracks from a last.fm playlist.
  search    Search for content using a specific source.
  url       Download content from URLs.
```

How about rip --quality url ... ?

**Comment by Gabri04** [comment]
> -m -c options work in version 1.9.7

Hello, I tried using the 1.9.7 version but it doesn't work. Probably outdated. Any suggestion?

**Comment by unMichalBak** [comment]
I can confirm the same problem with --code & --quality switches not working on Windows

**Comment by unMichalBak** [comment]
yes, I know that's why it's reported as a bug :) 
but I can confirm that for example : rip -c mp3 url <ACTUALURL>  
works correctly, also rip --codec mp3 url <ACTUALURL>  
so it seems that docs/manual is providing incorrect info or the order of args has changed

**Comment by tannerbaum** [comment]
> so it seems that docs/manual is providing incorrect info or the order of args has changed

The manual page found at `rip url --help` is up to date, the README and in turn github home page is out of date. What is confusing the situation is that users who are on older versions of the library will tell you these commands work fine. But indeed, --quality has been replaced with --max-quality.

**Comment by mortalis13** [comment]
Or I'm missing something or it's really confusing...
First time using it, so I may be wrong, but...it seems very simple:
`rip --help` tells about options `--quality` and `--codec`
`rip url --help` has no options, apart from `--help`

Tested installing a version from `pip` and from the `dev` branch.

So `--quality` and `--codec` are options for `rip` directly, that go before `url` or any other command.

There's no `--max-quality`, neither before `url` not after. It's not in the code either.  
Or it's a future idea to be changed in the code or a misdescription in the readme...

And you can use the options with and without a `=`.
Basically, just follow the help text from the binary, it tells the correct options that were configured in the code.
